components/mediation/mediators/oauth-mediator/org.wso2.micro.integrator.mediator.oauth/src/main/java/org/wso2/micro/integrator/mediator/oauth/MediatorException.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2009, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package org.wso2.micro.integrator.mediator.oauth;
+
+/**
+ * Exception for the sequence editor mediator service.
+ */
+@SuppressWarnings({"UnusedDeclaration"})
+public class MediatorException extends RuntimeException {
+
+    public MediatorException() {

```

Comment:
Do we need this constructor?

Reply:
Removed MediarException class. Refactored oauth mediator to throw SynapseException in 563f10f6e2d2e0936ec3843f9ae35d84a369bef9